app/s3_client/s3_csv_client.py

```diff
@@ -28,17 +28,8 @@ def get_csv_upload(service_id, upload_id):
     return get_s3_object(*get_csv_location(service_id, upload_id))
 
 
-def remove_blank_lines(filedata):
-    # sometimes people upload files with hundreds of blank lines at the end
-    data = filedata["data"]
-    cleaned_data = "\n".join(line for line in data.splitlines() if line.strip())
```

Comment:
Awesome job uncovering it was this method that was causing the variable substitution to not come through!

This line is the actual issue and causing the problem - it turns out we need to preserve the `\r\n` formatting for line endings, so if we just modify this line instead to look like this:

```python
cleaned_data = "\r\n".join(line for line in data.splitlines() if line.strip())
```

The variable interpolation starts working again.  It's unclear why we still need the `\r`, there's something else at play here, but I'm not sure that's worth spending the time digging into.